src/styles/_gif-display.scss

```diff
@@ -0,0 +1,35 @@
+.gif-box{
+    padding-top:20px;
+    display:flex;
+    flex-direction: row;
```

Comment:
flex-direction: row is not necessary because that is flexbox's default direction. should work without this style.